tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/DevOpsService.cs

```diff
@@ -91,6 +90,10 @@ public interface IDevOpsService
         public Task<Build> GetPipelineRun(int buildId);
         public Task<string> GetSDKPullRequestFromPipelineRun(int buildId, string language, int workItemId);
         public Task<bool> AddSdkInfoInReleasePlan(int workItemId, string language, string sdkGenerationPipelineUrl, string sdkPullRequestUrl);
+        public Task<bool> UpdateReleasePlanSDKDetails(int workItemId, List<SDKInfo> sdkLanguages, string languageExclusionNote = "");
+        public Task<bool> UpdateApiSpecStatus(int workItemId, string status);
+        public Task<bool> UpdateSpecPullRequest(int releasePlanWorkItemId, string specPullRequest);
```

Comment:
nit: we should be consistent with the variable name for the work item id. In the last 2 methods it is `releasePlanWorkItemID`, while the above ones is `workItemId`